docs/contributing/test-data-models-overview.md

```diff
@@ -0,0 +1,250 @@
+# Test data models overview
+
+A test data model is set of helper objects commercetools API consumers can use during writing their own tests in order to mock the data returned by our services.
+
+These helpers are flexible enough to represent different types of responses from different APIs and with different values.
+
+## Public API
+
+Each test data model has the same API which consists on:
+
+- A Rest builder
+- A Graphql builder
```

Comment:
Nit when writing we should use the "proper capitalization".

```suggestion
- A GraphQL builder
```

Reply:
Updated here: 620a0125473afc82bd57cb988907d7ff8e8aa740